WalletWasabi/Microservices/ProcessBridge.cs

```diff
@@ -77,11 +78,16 @@ public Process Start(string arguments, bool openConsole)
 			}
 		}
 
-		public async Task<(string response, int exitCode)> SendCommandAsync(string arguments, bool openConsole, CancellationToken cancel)
+		public async Task<(string response, int exitCode)> SendCommandAsync(string arguments, bool openConsole, CancellationToken cancel, Action<StreamWriter> standartInputWriter = null)
 		{
 			int exitCode;
+			bool redirectStandardInput = standartInputWriter is { };
 
-			using var process = Start(arguments, openConsole);
+			using var process = Start(arguments, openConsole, redirectStandardInput);
+			if (redirectStandardInput)
+			{
+				standartInputWriter(process.StandardInput);
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.standardinput?view=netcore-3.1#examples 

```csharp

                // End the input stream to the sort command.
                // When the stream closes, the sort command
                // writes the sorted text lines to the
                // console.
                myStreamWriter.Close();
```

-> In the example, `myProcess.StandardInput` is actually closed. In your code, I can see only that you flush `StandardInput` but I don't see a place where you close the stream.

Reply:
That is not necessary.

Reply:
Close calling the Flush but additionally not further write possible on the stream. So it should be good what we have now however I prefer to do exactly what is in the example.